### DISCUSSION TITLE
jest-mock-extended super slow on large projects

### DISCUSSION THREAD

**Discussion by sepehr500** [prisma/prisma#16915] [Q&A]
Hi. I have a pretty big prisma schema, so when I use `mockDeep` from jest-mock-extended, it takes 5 seconds or more for my ide to show any intelisense. Anybody have any workarounds for this? Anybody run into this?

**Comment by pantharshit00** [comment]
Hello!

Can you tell me how many lines does your schema contain so that we may try to reproduce this? In any case, I think opening a bug report in `jest-mock-extended` repository will make more sense here if problem is specific to that package.

**Reply by sepehr500** [reply]
Hello. My schema file is 7k lines. Just trying to see if anybody has schema of similar size, and is running into issues.

**Reply by pantharshit00** [reply]
Okay! Last thing I would like to ask if you can set `DEBUG=true` in environment variables and paste the output here. That might give us an idea on which step is actually taking long time.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
